ui/StatusQ/src/StatusQ/Core/LocaleUtils.qml

```diff
@@ -175,6 +175,8 @@ QtObject {
         // Add symbol
         if (currencyAmount.symbol && !optNoSymbol) {
             amountStr = "%1%2 %3".arg(amountStr).arg(amountSuffix).arg(currencyAmount.symbol)
+        } else {
+            amountStr = "%1%2".arg(amountStr).arg(amountSuffix)
```

Comment:
hmm careful with this, "optNoSymbol" is used whenever we want this to output a pure number (no symbol, no suffixes, no nothing). Could you make sure this change couldn't break something where this setting is used? (I'm thinking about the Send Modal when the user switches between Fiat and Crypto input mode)